### PR TITLE
chore(api): update api dependencies for the cli

### PR DESCRIPTION
**Pull request by sundersc**
Updates the api packages dependencies for the cli with the latest versions.

### CODE REVIEW

File: packages/amplify-category-function/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "folder-hash": "^4.0.2",
     "fs-extra": "^8.1.0",
     "globby": "^11.0.3",
-    "graphql-transformer-core": "^7.6.1",
```
Comment (sachscode) [inline packages/amplify-category-function/package.json:36]: question: Is this change required? wouldn't ^7.6.1 pull in ^7.6.2 automatically on publish?

**Review by sachscode — COMMENTED** [review]
LGTM, just one general question.

File: packages/amplify-category-function/package.json (hunk 1 above)
Comment (sundersc) [reply packages/amplify-category-function/package.json:36]: Technically, yes. But we generate this PR using a script which updates the version explicitly to keep it consistent across the repository.

Comment (sachscode) [reply packages/amplify-category-function/package.json:36]: Thanks

**Review by sachscode — APPROVED** [review]

**Review by AaronZyLee — APPROVED** [review]

**Review by danielleadams — APPROVED** [review]
